launcher/ui/pages/instance/InstanceSettingsPage.cpp

```diff
@@ -74,7 +78,7 @@ InstanceSettingsPage::InstanceSettingsPage(BaseInstance *inst, QWidget *parent)
 
 bool InstanceSettingsPage::shouldDisplay() const
 {
-    return !m_instance->isRunning();
+    return true;
 }
```

Comment:
It should behave the same if you remove it altogether.

Reply:
So... Should I remove this entire function?

Reply:
Oh, it's a virtual function and it's already implemented by BasePage so no need to remove calls (in fact that would not be what you want)